src/lib/config.js

```diff
@@ -212,7 +212,8 @@ function getLocalConfig () {
   // The spread is added to every quoted rate
   const fxSpread = Number(Config.getEnv(envPrefix, 'FX_SPREAD')) || DEFAULT_FX_SPREAD
 
```

Comment:
Could you do the same thing to `fxSpread` as you did for `slippage`? I'm running into an error now that this ignores what you set it `CONNECTOR_FX_SPREAD`, but if you change this to
```
const fxSpreadString = Config.getEnv(envPrefix, 'FX_SPREAD') 
const fxSpread = fxSpreadString ? Number(fxSpreadString) : DEFAULT_FX_SPREAD
```
it works as expected